driver.py

```diff
@@ -269,7 +271,7 @@ def compute_effective_available_space(self, torrent_list):
         return effective_available_size
 
     def filter_out_managed_items_already_in_client(
-        self, managed_group, incomplete_group, complete_group
+            self, managed_group, incomplete_group, complete_group
     ):
```

Comment:
I'm not a fan of this particular style.  Although I notice that this is indeed how PEP8 specifies argument lists should be indented...  There's a big thread on the `black` issue about this here, seems controversial: https://github.com/psf/black/issues/1178

Is it possible to just drop this particular correction (indenting argument lists in definition by 8 spaces)?  I know it's technically a PEP8 violation but I find the 8 space variant really ugly.

Reply:
Yep. You are right. I'll fix this.

Reply:
Resolved in aab4917.